demos/defender/defender_demo_json/CMakeLists.txt

```diff
@@ -12,12 +12,11 @@ include( ${CMAKE_SOURCE_DIR}/libraries/standard/coreJSON/jsonFilePaths.cmake )
 # Include Defender library's source and header path variables.
 include( ${CMAKE_SOURCE_DIR}/libraries/aws/device-defender-for-aws-iot-embedded-sdk/defenderFilePaths.cmake )
 
+file( GLOB DEMO_SRCS "*.c*" )
```

Comment:
```suggestion
# Find all C and CPP source files (
# CPP files are searched for supporting CI build checks that verify C++ build compliance of librariesin C++ builds)
file( GLOB DEMO_SRCS "*.c*" )
```

Reply:
Similar comment in all other CMake files would be helpful

Reply:
Added